test/unit/lodash.spec.js

```diff
@@ -0,0 +1,568 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function () {
+    var params;
+
+    describe('camelCase', function () {
+        before(function () {
+            params = datasets.camelCase();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length(1);
+        });
+        it('should convert string to camel case', function () {
+                expect(_.camelCase.apply(_, params)).to.eql('someStringToTestFunction');
```

Comment:
In my opinion, without looking to this file, that solution is impossible to reveal